src/build/render/components/bits.js

```diff
@@ -20,17 +21,19 @@ const html = commonTags.stripIndent(commonTags.html);
 
 const classes = (classArr) => classArr.length > 0 ? `class="${classArr.join(" ")}"` : "";
 
+const localizer = R.curry((bundle, lang, key) => bundle[key][lang]);
```

Comment:
`R.curry` takes a function and returns a version where calling it with just some of the arguments returns a function of the remaining. For example:

```js
let original = (a, b, c) => {...};
let curried = R.curry(original);
curried(1)(2)(3) === original(1, 2, 3)
```

This is useful when you want to pass around or initialize a function which has some of its arguments pre-set but want to reserve the ability to use its original form when needed.